price-estimator/src/filter.rs

```diff
@@ -54,6 +62,9 @@ async fn handle_rejection(err: Rejection) -> Result<impl Reply, Infallible> {
     } else if let Some(NoTokenInfo) = err.find() {
         code = StatusCode::BAD_REQUEST;
         message = "request with atoms=true for token we don't have erc20 info for";
```

Comment:
Unrelated: It seems like this message is a) inverted and b) referencing the deprecated parameter